frame/staking/src/tests.rs

```diff
@@ -153,7 +153,7 @@ fn basic_setup_works() {
 		// for these stash ids, see
 		// https://github.com/paritytech/substrate/
 		//   blob/631d4cdbcad438248c2597213918d8207d85bf6e/frame/staking/src/mock.rs#L435-L441
```

Comment:
```suggestion
```

Reply:
If we keep this comment it won't age well, but I will admit it was helpful when I was working on tests